src/fparser/two/tests/fortran2003/test_primary_r701.py

```diff
@@ -145,16 +145,35 @@ def test_array_constructor(f2003_create):
         expected_str='[1.2, 2.3E+2, - 5.1E-3]')
 
 
+@pytest.mark.xfail(reason="Matches Designator before Structure Constructor "
+                   "(issue #284)")
 def test_structure_constructor(f2003_create):
```

Comment:
Since you've touched this test, please could you fix the pylint warning about `f2003_create` being unused by using the @pytest.mark.usefixtures().

Reply:
updated.